local-cluster/tests/local_cluster.rs

```diff
@@ -5738,7 +5738,6 @@ fn test_randomly_mixed_block_verification_methods_between_bootstrap_and_not() {
 
 /// Forks previous marked invalid should be marked as such in fork choice on restart
 #[test]
-#[ignore]
```

Comment:
Does this change specifically make this test less flaky / passable ?

Reply:
it does but we don't know why exactly yet. See: https://github.com/anza-xyz/agave/pull/823#discussion_r1575382916. so i need to debug